tools/system_libs.py

```diff
@@ -477,11 +481,15 @@ def do_create():
       force = force.union(deps)
   ret.sort(key=lambda x: x.endswith('.a')) # make sure to put .a files at the end.
 
-  # Handle backend compiler_rt separately because it is not a bitcode system lib like the others.
-  # Here, just ensure that it's in the cache.
+  # Handle backend compiler_rt separately because it is not a bitcode system lib
+  # like the others.  Here, just ensure that it's in the cache.
   if shared.Settings.BINARYEN and shared.Settings.WASM_BACKEND:
-    shared.Cache.get('wasm_compiler_rt.a', lambda: create_wasm_compiler_rt('wasm_compiler_rt.a'), extension='a')
-    shared.Cache.get('wasm_libc_rt.a', lambda: create_wasm_libc_rt('wasm_libc_rt.a'), extension='a')
+    if shared.Settings.WASM_BACKEND == 2:
+      shared.Cache.get('libwasm_compiler_rt.a', lambda: create_wasm_compiler_rt('libwasm_compiler_rt.a'), extension='a')
```

Comment:
is this just because the `lib` prefix is necessary for lld? if the contents are identical, could we just always call it the `lib*` name?